package.json

```diff
@@ -33,7 +33,8 @@
     "express-prismic": "^1.1.1",
     "graphql": "^0.6.0",
     "lodash": "^4.13.1",
```

Comment:
That's what I thought, but it crashes without it.

![2016-07-12-122133_1382x884_scrot](https://cloud.githubusercontent.com/assets/6134406/16765312/3af64948-482b-11e6-8824-457ce76155ac.png)